docs/_docs/array-accumulators.md

```
@@ -2,9 +2,20 @@
 title: Array accumulators
 subtitle: Accumulators applicable to arrays
 tags: [accumulators, array]
-keywords: [count, first, last, max, min, sum] # AUTO-GENERATED KEYWORDS
+keywords: [any, count, every, first, last, max, min, sum] # AUTO-GENERATED KEYWORDS
 ---
+
+`every` and `any` accumulate boolean values directly and take no predicate parameter. They are distinct from the `all(...)` and `none(...)` array predicates, which evaluate a supplied predicate against array elements.
+
 <!-- START AUTO-GENERATED -->
+##### any
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h5

(MD001, heading-increment)